hw/ip/edn/data/edn_testplan.hjson

```diff
@@ -24,10 +23,49 @@
       tests: ["edn_smoke"]
     }
     {
-      name: feature1
-      desc: '''Add more test entries here like above.'''
-      milestone: V1
+      name: firmware
+      desc: '''
+            Verify regen bit enables/disables write access to control registers
+            Verify firmware generate command causes entropy to be distributed to hw agents.
+            Verify cmd_fifo_reset bit causes fifos to reset.
+            '''
+      milestone: V2
       tests: []
     }
+    {
+      name: csrng_cmd
+      desc: '''
+            Verify endpoint agent reqs generate csrng commands as predicted.
+            Verify auto-generate mode behaves as predicted.
+            Verify max_num_reqs_between_reseeds in auto-generate mode.
+            Verify boot-time request mode behaves as predicted.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: genbits
+      desc: '''
+            Verify genbits input is transferred to endpoint as predicted.
+            Verify fips bit(s) are properly transferred to endpoint.
```

Comment:
Verify that multiple endpoints can request for entropy, and in random order of requesting.